src/pybpmn/parser.py

```diff
@@ -335,10 +348,10 @@ def _edge_to_anns(edge: Element, model_element: Element, id_to_shape_ann: Dict[s
         if rel not in attrib:
             continue
         sid = attrib[rel]
-        ann = id_to_shape_ann.get(sid, None)
+        ann = id_to_ann.get(sid, None)
         if ann is None and category == syntax.ASSOCIATION:
-            # TODO implement that associations can be connected to other edges (e.g. sequenceFlow)
-            continue
+            # TODO implement that associations can be connected to other associations
+            raise InvalidBpmnException(f"Association {sid} has another association as src or target!")
```

Comment:
InvalidBpmnException hat 2 Argumente, den error_type und die (optionalen) details. Die Idee ist dass man nach dem error_type groupieren kann (wie in dem sap_sam notebook). Das geht jedoch nciht wenn sowas wie die sid im error_type steht. Ich würde es daher so anpassen:
`InvalidBpmnException("Association has another association as src or target", sid)`